### PR TITLE
fix(CI): bump request_timeout for 1-org

### PR DESCRIPTION
**Pull request by apeabody**
- bump `request_timeout` for 1-org

Fixes: #1069

### CODE REVIEW

File: 1-org/modules/centralized-logging/providers.tf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  Provider request timeout configuration
+ *****************************************/
+provider "google" {
```
Comment (daniel-cit) [inline 1-org/modules/centralized-logging/providers.tf:20]: @apeabody 
I think this file need to be created at the `envs/shared` folder
https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/1-org/envs/shared
like in the usages that we have in the 3-network step

Comment (apeabody) [reply 1-org/modules/centralized-logging/providers.tf]: Sure - I've moved the file, and backed out the other changes so we can focus on this hypothesis.

**Comment by apeabody** [comment]
```
Step #16 - "converge-networks": Error: Error creating AccessLevel: googleapi: Error 404: Policy not found
Step #16 - "converge-networks": 
Step #16 - "converge-networks":   with module.base_env.module.restricted_shared_vpc.module.access_level_members.google_access_context_manager_access_level.access_level,
Step #16 - "converge-networks":   on .terraform/modules/base_env.restricted_shared_vpc.access_level_members/modules/access_level/main.tf line 21, in resource "google_access_context_manager_access_level" "access_level":
Step #16 - "converge-networks":   21: resource "google_access_context_manager_access_level" "access_level" {
Step #16 - "converge-networks": }
```

**Review by bharathkkb — APPROVED** [review]

**Review by daniel-cit — APPROVED** [review]
LGTM

**Comment by daniel-cit** [comment]
@apeabody build is green 🎉

**Comment by daniel-cit** [comment]
@apeabody this will help with the other builds :)

**Pull request merged**